### PR TITLE
Update to add orgs

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Adding orgs

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/validator/GenerateUrlRequestValidatorImpl.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ private boolean isNewSubmission(InitialPackage initialPackage) {
      */
     private List<String> validateParties(InitialPackage initialPackage) {
 
-        if (StringUtils.isBlank(initialPackage.getCourt().getFileNumber()) && initialPackage.getParties().isEmpty()) {
+        if (StringUtils.isBlank(initialPackage.getCourt().getFileNumber()) && (initialPackage.getParties() == null || initialPackage.getParties().isEmpty())) {
```
Comment (KDNeufeld) [inline src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/validator/GenerateUrlRequestValidatorImpl.java:127]: or ... `&& (CollectionUtils.isEmpty(initialpackage.getParties())`

Comment (TayGov) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/validator/GenerateUrlRequestValidatorImpl.java:127]: Yes, this method is being updated in FLA-895. Which I'm working on now

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/TestHelpers.java [hunk 2]
```diff
@@ -130,19 +132,21 @@ public static Court createCourt() {
...
     public static FilingPackage createPackage(
             Court court,
             List<Document> documents,
-            List<Party> parties) {
+            List<Individual> parties,
+            List<Organization> organizations) {
         return FilingPackage.builder()
                 .court(court)
                 .documents(documents)
                 .parties(parties)
```
Comment (KDNeufeld) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/TestHelpers.java:148]: This all looks really good.  My only comment is regarding the refactoring that split Parties into Individuals and Organizations - there could be more work here if you deemed appropriate.  You may want to consider keeping the Party term to the interface and use Individual and Organization as the impls. 
i.e, 
line 143 could be `List<Individual> individuals`
line 148 could be `.individuals(individuals)`

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/models/FilingPakageTest.java [hunk 3]
```diff
@@ -95,21 +102,32 @@ private void assertCourt(FilingPackage actual) {
         Assertions.assertEquals(PARTICIPATING_CLASS, actual.getCourt().getParticipatingClass());
     }
 
-    private List<Party> getParties() {
-        List<Party> parties = new ArrayList<>();
-        Party party = new Party(
-                PARTY_TYPE_CD,
+    private List<Individual> getParties() {
```
Comment (KDNeufeld) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/models/FilingPakageTest.java:105]: possibly more refactoring here
`private List<Individual> getIndividuals() {`

File: src/backend/libs/efiling-bom/pom.xml [hunk 4]
```diff
@@ -254,6 +255,11 @@
                 <artifactId>janino</artifactId>
                 <version>${org.codehaus.janino}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-collections4</artifactId>
+                <version>${org.apache.commons.collections.collections.version}</version>
+            </dependency>
```
Comment (KDNeufeld) [inline src/backend/libs/efiling-bom/pom.xml:262]: Nice. Yeah, I find this library useful.

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/TestHelpers.java (hunk 2 above)
Comment (TayGov) [reply src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/TestHelpers.java:148]: I agree the reason I left the parties name was is the naming we have to use in the swagger. So I left it named that we throughout. Same as below comment.

File: src/backend/libs/efiling-bom/pom.xml (hunk 4 above)
Comment (TayGov) [reply src/backend/libs/efiling-bom/pom.xml:262]: Yes the apache commons are great utils!

File: src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/Individual.java [hunk 5]
```diff
@@ -3,9 +3,8 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class Party {
+public class Individual {
```
Comment (KDNeufeld) [inline src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/Individual.java:6]: Nice.  Yeah, this was a necessary change but large.
Fantastic work!

**Review by alexjoybc — COMMENTED** [review]
Awesome change @TayGov , the only thing is to update docsify, do you need help with it ?

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**